tests/test_purgatory.py

```diff
@@ -68,18 +67,21 @@ def test_purgatory_with_assets(client, base_ddo_url, events_object, monkeypatch)
 
     purgatory = PurgatoryForTesting(events_object._es_instance)
     published_ddo = get_ddo(client, base_ddo_url, did)
-    assert published_ddo["purgatory"]["state"] == "false"
+    assert not published_ddo["purgatory"]["state"]
+    isinstance(published_ddo["purgatory"]["state"], bool)
```

Comment:
you can combine these two asserts, try: assert ... is False and assert ... is True. The `is` keyword will check both that it is boolean and its truthness value.

Reply:
this is applicable to all the double-assertions in this file